student/serializers.py

```diff
@@ -65,4 +66,5 @@ class Meta:
             'LoginToken',
             'LoginHash',
             'timeAcceptedTos',
+            'favorite_num'
         )
```

Comment:
This is all that was technically needed for the serializer to work with the new field :)